internal/driver/glfw/window.go

```diff
@@ -279,7 +279,7 @@ func (w *window) destroy(d *gLDriver) {
 	w.DestroyEventQueue()
 	cache.CleanCanvas(w.canvas)
 
-	if w.master {
+	if w.master || len(w.driver.windowList()) == 1 {
```

Comment:
== 1 is only exit criteria on a desktop app with system tray turned on isn't it?

Reply:
Hmm. I though it would be no different than the check that quit if there were 0 windows. Will have to look into that.

Reply:
I moved it back to where it was